frontend/src/component/admin/instance-privacy/InstancePrivacySection.tsx

```diff
@@ -0,0 +1,211 @@
+    marginTop: theme.spacing(1),
+    padding: theme.spacing(1),
+    border: `1px solid ${theme.palette.divider}`,
+    borderRadius: theme.shape.borderRadius,
+    borderWidth: 1,
+    wordWrap: 'break-word',
+    whiteSpace: 'pre-wrap',
+    fontFamily: 'monospace',
+    lineHeight: 1.5,
+}));
+
+const ToolTipDescriptionText = styled('p')(({ theme }) => ({
+    color: theme.palette.text.primary,
+    fontSize: theme.fontSizes.smallBody,
+    marginTop: theme.spacing(1),
+}));
+
+export const InstancePrivacySection = ({
+    title,
+    infoText,
+    concreteDetails,
+    enabled,
+    changeInfoText,
+    variablesText,
+    dependsOnText,
+}: IPrivacyProps) => {
+    return (
+        <StyledContainer>
+            <StyledCardTitleRow>
+                <b>{title}</b>
+                <StyledDataCollectionBadge>
+                    <ConditionallyRender
+                        condition={enabled}
+                        show={
+                            <Badge color="success" icon={<CheckIcon />}>
+                                Data is collected
+                            </Badge>
+                        }
+                    />
+                    <ConditionallyRender
+                        condition={!enabled}
+                        show={
+                            <Badge color="neutral" icon={<ClearIcon />}>
+                                No data is collected
+                            </Badge>
+                        }
+                    />
+                </StyledDataCollectionBadge>
+            </StyledCardTitleRow>
+
+            <StyledCardDescription>
+                <StyledDataCollectionPropertyTable>
+                    <StyledDataCollectionExplanation>
+                        {infoText}
+                    </StyledDataCollectionExplanation>
+                    <StyledDataCollectionPropertyCell>
+                        <StyledTag>
+                            <TooltipLink
+                                tooltip={
+                                    <ToolTipInstructionContent
+                                        changeInfoText={changeInfoText}
+                                        variablesText={variablesText}
+                                        dependsOnText={dependsOnText}
+                                    />
+                                }
+                            >
+                                {enabled
+                                    ? 'How to disable collecting data?'
+                                    : 'How to enable collecting data?'}
+                            </TooltipLink>
+                        </StyledTag>
+                    </StyledDataCollectionPropertyCell>
+                </StyledDataCollectionPropertyTable>
+
+                <StyledDataCollectionPropertyTable>
+                    {Object.entries(concreteDetails).map(([key, value]) => {
+                        return (
+                            <StyledDataCollectionPropertyRow>
```

Comment:
Each child component here should have a unique key prop, otherwise we get an ugly warning on the browser console.

If `key` is unique, then it could be something like:
```suggestion
                            <StyledDataCollectionPropertyRow key={key}>
```